docker/dockerfiles/Dockerfile.onnx.cpu

```diff
@@ -43,13 +44,18 @@ RUN pip3 install --upgrade pip  && pip3 install \
     && rm -rf ~/.cache/pip
 
 FROM scratch
+ARG TARGETPLATFORM
+
 COPY --from=base / /
 
 WORKDIR /build
 COPY . .
 RUN make create_wheels
 RUN pip3 install dist/inference_cli*.whl dist/inference_core*.whl dist/inference_cpu*.whl dist/inference_sdk*.whl
 
+RUN if [ "${TARGETPLATFORM}" = "linux/amd64" ]; then pip3 install -r requirements/requirements.vino.txt; rm -rf ~/.cache/pip; fi
```

Comment:
Do we need to uninstall onnxruntime before we install requirements/requirements.vino.txt? Wondering if we'll have two conflicting versions of onnxruntime if we don't explicitly uninstall it first.

Reply:
no, both are visible. confirmed manually

Reply:
```
(base) ppeczek@pawel-dev:~/inference$ docker run -it --entrypoint bash roboflow/roboflow-inference-server-cpu:dev
root@e30729664123:/app# python
Python 3.9.18 (main, Nov 29 2023, 05:56:31)
[GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import onnxruntime
>>> onnxruntime.get_available_providers()
['OpenVINOExecutionProvider', 'CPUExecutionProvider']
```